doc/Development_Documentation/18_Tools_and_Features/39_Sitemaps.md

```diff
@@ -16,7 +16,7 @@ for details).
 
 ```yaml
 PrestaSitemapBundle:
-    resource: "@PrestaSitemapBundle/Resources/config/routing.yml"
+    resource: "@PrestaSitemapBundle/Resources/config/routing.yaml"
```

Comment:
```suggestion
    resource: "@PrestaSitemapBundle/Resources/config/routing.yml"
```

This is still `yml`: https://github.com/prestaconcept/PrestaSitemapBundle/tree/2.x/Resources/config

Reply:
good catch! thanks 😄